Sources/GRPCCore/Internal/_MethodConfigurations.swift

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+// TODO: when swift(>=5.9), use 'package' access level
```

Comment:
> Aren't we already using other 5.9+-only features in the v2 code? 

I don't think so

> Wouldn't the availability guard allow us to use this?

Yes, but we'd still need to do what we're already doing for older versions so we end up maintaining two code paths rather than one. Easier to just switch wholesale when 5.9 is our minimum version.